README.md

```diff
@@ -1,2 +1,145 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello_microverse_project] <a name="about-project"></a>
+
+**[hello_microverse_project]** is a project that it's main goal is to master how to use git and GitHub and other tools learned earlier.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project is built using the client side technologies
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">Html</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp">CSS</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Follow the live demo link to the deployed site.
+
+- [Live Demo Link](https://mpdaddy.github.io/hello_microverse_project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+You need to have installed git in your local system and you also need a code editor.
+
+### Setup
+Go straight to the repository and click the code button and choose the url.
+you can use https link or the ssh one.
+
+
+### Install
+
+Clone this repository to your desired folder:
+using 'git clone 'command and pasting the link you copied.
+
+
+### Usage
+
+To run the project, execute the following command:
+navigate to the project using the command  "cd 'project name'"
+and then run
+'code .'
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@MPDADDY](https://github.com/MPDADDY)
+- Twitter: [@anoldmwawasi](https://twitter.com/anold_mwawasi)
+- LinkedIn:[anoldmwawasi](https://www.linkedin.com/in/anold-mwawasi/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+For now as it is an intro project , features will be communicated in the near future.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+>If you like this project your may want to add a star on my git hub account i'll really appreciate it.
```

Comment:
- [ ] **[OPTIONAL]** This is a muted text, it looks like a description, I would recommend you remove the greater than sign ">" and just write out the text.

- [ ] **[OPTIONAL]** Some of your commit messages are not well written. They should be imperative and in the present tense. Look at the commit messages below. Instead of:

  ```
  Added test.md file
  ```
  what you should have is:

  ```
  Add test.md file
  ```
  Going forward, I would recommend you take this rule into consideration.

  ![image](https://github.com/MPDADDY/hello_microverse_project/assets/36542986/00cf1835-c85a-4df9-9d26-947475e06d79)